server/routes/api.js

```diff
@@ -104,4 +105,29 @@ routes.get('/pairing/paired', (req, res) => {
   });
 });
 
+routes.get('/wishlist/:familyId/read', (req, res) => {
+
+  const familyId = req.params.familyId;
+  if ( !familyId.match(/[0-F]{24}/gi) ) {
+    res.status(200).json({ message: 'No family found'});
+  }
+
+  Wishlist.where({family: familyId})
+    .exec((err, wishlist) => {
+      if(err) {
+        res.status(500).json(err);
+      } else {
```

Comment:
If everything that isn't an error, we can just break the error if block here and have everything else not wrapped within this same if. I dont think anything else after the if block gets processed if we move into a return statement.

example: 

```javascript
if (err) {
  return res.status(500).json(err);
}

// rest of code
```